packages/big-design/src/components/Table/Table.tsx

```diff
@@ -138,7 +138,7 @@ const InternalTable = <T extends TableItem>(
           <DragIconHeaderCell
             actionsRef={actionsRef}
             headerCellIconRef={headerCellIconRef}
-            width={headerCellWidths.length ? headerCellWidths[0] : 'auto'}
+            width={headerCellWidths.length ? headerCellWidths[0] : '50px'}
```

Comment:
Instead of hardcoding this width, inside `/src/components/Table/HeaderCell`, what if we renamed `StyledTableHeaderCheckbox` to be more generic (something like `StyledTableHeaderIcon`) and reuse that component inside `DragIconHeaderCell` instead?

Reply:
Oh that's brilliant, a much more reusable solution! Pushing those changes shortly